integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiJmsRecovery.java

```diff
@@ -0,0 +1,739 @@
+         .append("/management/wls/latest/datasources/id/")
+         .append(resourcesName)
+         .append("/")
+         .append(" --silent --show-error ");
+    logger.info("checkJdbcRuntime: curl command {0}", new String(curlString));
+    try {
+      result = exec(new String(curlString), true);
+    } catch (Exception ex) {
+      logger.info("checkJdbcRuntime: caught unexpected exception {0}", ex);
+      return null;
+    }
+    return result;
+  }
+
+  private static void createJobToChangePermissionsOnPvHostPath(String pvName, String pvcName, String namespace) {
+    logger.info("Running Kubernetes job to create domain");
+    V1Job jobBody = new V1Job()
+        .metadata(
+            new V1ObjectMeta()
+                .name("change-permissions-onpv-job-" + pvName) // name of the job
+                .namespace(namespace))
+        .spec(new V1JobSpec()
+            .backoffLimit(0) // try only once
+            .template(new V1PodTemplateSpec()
+                .spec(new V1PodSpec()
+                    .restartPolicy("Never")
+                    .addContainersItem(
+                        createfixPVCOwnerContainer(pvName,
+                        "/shared")) // mounted under /shared inside pod
+                    .volumes(Arrays.asList(
+                        new V1Volume()
+                            .name(pvName)
+                            .persistentVolumeClaim(
+                                new V1PersistentVolumeClaimVolumeSource()
+                                    .claimName(pvcName))))
+                    .imagePullSecrets(Arrays.asList(
+                        new V1LocalObjectReference()
+                            .name(BASE_IMAGES_REPO_SECRET)))))); // this secret is used only for non-kind cluster
+
+    String jobName = createJobAndWaitUntilComplete(jobBody, namespace);
+
+    // check job status and fail test if the job failed
+    V1Job job = assertDoesNotThrow(() -> getJob(jobName, namespace),
+        "Getting the job failed");
+    if (job != null) {
+      V1JobCondition jobCondition = job.getStatus().getConditions().stream().filter(
+          v1JobCondition -> "Failed".equalsIgnoreCase(v1JobCondition.getType()))
+          .findAny()
+          .orElse(null);
+      if (jobCondition != null) {
+        logger.severe("Job {0} failed to change permissions on PV hostpath", jobName);
+        List<V1Pod> pods = assertDoesNotThrow(() -> listPods(
+            namespace, "job-name=" + jobName).getItems(),
+            "Listing pods failed");
+        if (!pods.isEmpty()) {
+          String podLog = assertDoesNotThrow(() -> getPodLog(pods.get(0).getMetadata().getName(), namespace),
+              "Failed to get pod log");
+          logger.severe(podLog);
+          fail("Change permissions on PV hostpath job failed");
+        }
+      }
+    }
+  }
+
+  private void checkLogsOnPV(String commandToExecuteInsidePod, String podName) {
```

Comment:
looks like this is not used, can be removed

Reply:
removed